cli/create-user.php

```diff
@@ -1,38 +1,93 @@
 #!/usr/bin/env php
 <?php
 declare(strict_types=1);
+require(__DIR__ . '/_cli.php');
 
-$isUpdate = false;
-require(__DIR__ . '/_update-or-create-user.php');
+$cliOptions = new class extends CliOptionsParser {
+	public string $user;
+	public string $password;
+	public string $apiPassword;
+	public string $language;
+	public string $email;
+	public string $token;
+	public int $purgeAfterMonths;
+	public int $feedMinArticles;
+	public int $feedTtl;
+	public int $sinceHoursPostsPerRss;
+	public int $maxPostsPerRss;
+	public bool $noDefaultFeeds;
 
-$username = $GLOBALS['options']['valid']['user'];
-if (!FreshRSS_user_Controller::checkUsername($username)) {
-	fail('FreshRSS error: invalid username “' . $username .
-		'”! Must be matching ' . FreshRSS_user_Controller::USERNAME_PATTERN);
-}
+	public function __construct() {
+		$this->addRequiredOption('user', (new CliOption('user')));
+		$this->addOption('password', (new CliOption('password')));
+		$this->addOption('apiPassword', (new CliOption('api-password'))->deprecatedAs('api_password'));
+		$this->addOption('language', (new CliOption('language')));
+		$this->addOption('email', (new CliOption('email')));
+		$this->addOption('token', (new CliOption('token')));
+		$this->addOption(
+			'purgeAfterMonths',
+			(new CliOption('purge-after-months'))->typeOfInt()->deprecatedAs('purge_after_months')
+		);
+		$this->addOption(
+			'feedMinArticles',
+			(new CliOption('feed-min-articles-default'))->typeOfInt()->deprecatedAs('feed_min_articles_default')
+		);
+		$this->addOption(
+			'feedTtl',
+			(new CliOption('feed-ttl-default'))->typeOfInt()->deprecatedAs('feed_ttl_default')
+		);
+		$this->addOption(
+			'sinceHoursPostsPerRss',
+			(new CliOption('since-hours-posts-per-rss'))->typeOfInt()->deprecatedAs('since_hours_posts_per_rss')
+		);
+		$this->addOption(
+			'maxPostsPerRss',
+			(new CliOption('max-posts-per-rss'))->typeOfInt()->deprecatedAs('max_posts_per_rss')
+		);
+		$this->addOption(
+			'noDefaultFeeds',
+			(new CliOption('no-default-feeds'))->withValueNone()->deprecatedAs('no_default_feeds')
+		);
+		parent::__construct();
+	}
+};
 
-$usernames = listUsers();
-if (preg_grep("/^$username$/i", $usernames)) {
-	fail('FreshRSS warning: username already exists “' . $username . '”', EXIT_CODE_ALREADY_EXISTS);
```

Comment:
Removing this test caused a regression, fixed in https://github.com/FreshRSS/FreshRSS/pull/6214